meteor/sofie-server/data-access/facades/repository-facade.ts

```diff
@@ -21,7 +22,13 @@ export class RepositoryFacade {
 			new MongoEntityConverter(),
 			this.createSegmentRepository()
 		)
-		return CachedRundownRepository.getInstance(mongoRundownRepository)
+
+		const mongoRundownPlaylistRepository: MongoRundownPlaylistRepository = new MongoRundownPlaylistRepository(
```

Comment:
Not sure if you should change this is or not, but we generally want to depend on abstractions instead of concrete classes. What that means here is that the type should be `RundownRepository` and not `MongoRundownPlaylistRepository`.

I'm on the fence whether you should change it or not since the purpose of this class is to instantiate the concrete implementation of a class and this variable will only ever be a `MongoRundownPlaylistRepository`.

I can see good arguments from both sides, so I will let you decide :)